app/_hub/kong-inc/ldap-auth/index.md

```diff
@@ -52,7 +52,7 @@ params:
   service_id: false
   route_id: true
   consumer_id: false
-  protocols: ["http", "https"]
+  protocols: ["http", "https", "gprc", "grpcs"]
```

Comment:
Need to add a gRPC example?

Reply:
i tried to; but there's no https example either, so it felt out of place.  the "usage" section shows the header format, and it's just the same for grpc